django_guid/signals.py

```diff
@@ -14,11 +14,12 @@ def delete_guid(sender: Optional[dict], **kwargs: dict) -> None:
     """
     Receiver function for when a request finishes.
 
-    When a request is finished, delete a requests _guid reference to prevent memory leaks.
+    When a request is finished the delete_guid method is called to remove the request _guid reference.
+    This is important to maintain, to prevent a memory leak.
```

Comment:
Not really a fan of this change, feels like it's more words to say the same thing?